source/layoutex/layout/ParagraphLayout.h

```diff
@@ -528,6 +528,9 @@ class U_LAYOUTEX_API ParagraphLayout : public UObject
      *              to zero, a <code>ParagraphLayout::Line</code> object representing the
      *              rest of the paragraph will be returned.
      *
+     * @param kind denotes the type of break iterator used when truncating line to fit in in width.
+     *             Default is UBRK_LINE in order to not introduce breaking changes
```

Comment:
the reasoning for the default shouldn't be in here -- just say that's the default as in with the stock ICU

Reply:
Done